src/gluonts/torch/modules/scaler.py

```diff
@@ -57,7 +67,10 @@ def forward(
         denominator = torch.max(
             total_observed, torch.ones_like(total_observed)
         )
-        default_scale = weighted_sum.sum(dim=0) / denominator
```

Comment:
I think this was wrong in the first place, mixing things across dimension 0 should not occur probably